tests/search.test.ts

```diff
@@ -992,6 +992,30 @@ describe.each([
       client.index(index.uid).search('prince', {}),
     ).rejects.toHaveProperty('cause.code', ErrorStatusCode.INDEX_NOT_FOUND);
   });
+
+  test(`${permission} key: Search with locales`, async () => {
+    const ind = (await getClient(permission)).index(index.uid),
+      masterInd = (await getClient('Master')).index(index.uid);
+
+    await masterInd.waitForTask(
+      (
+        await masterInd.updateSettings({
+          localizedAttributes: [
+            {
+              attributePatterns: ['title', 'comment'],
+              locales: ['fra', 'eng'],
+            },
+          ],
+        })
+      ).taskUid,
+    );
+
+    const response = await ind.search('french', {
+      locales: ['fra', 'eng'],
+    });
+
+    expect(response.hits.length).toEqual(2);
```

Comment:
I am a little unsure about what this feature does, so I copied [the test from the PHP repo](https://github.com/meilisearch/meilisearch-php/pull/662/files) and it seemed to work on my machine on 1.10 release candidate. Can you give me more details about how this feature works more exactly?

Reply:
Ok I've checked what's wrong, and it's a bit funny 😁
This test is put after one named `Try to search on deleted index and fail`, so by running the entire test file (like I did locally and like in the CI) the test fail because the index has previously been deleted 🙈
That's also why it worked on your side, I guess you tried to run only the test you added which is completely understandable. 
Anyway, if you move it before the test that deletes the index, it should be working 🎉

Reply:
I see. I always ran it without focus on other PRs, but somehow I wasn't patient enough for this one.
I also keep forgetting to remove the focus and pushing it, thankfully ESLint catches it.
Done!

Reply:
I feel you, the exact same thing always happens to me too 🤣